front/src/components/MailMerge/MailMergeHelpers.ts

```diff
@@ -19,4 +20,19 @@ export function registerHandlebarsHelpers() {
         return value
     }
   });
+  Handlebars.registerHelper('querystring', (value: string) => {
+    const linkAttributes =  useUrlParams()
+    switch (value) {
+      case 'hash':
+        return linkAttributes.hash;
+      case 'siteViewUrl':
+        return linkAttributes.siteViewUrl;
+      case 'pageViewUrl':
+        return linkAttributes.pageViewUrl
+      case 'q':
+        return linkAttributes.q
+      default:
+        return value
+    }

```

Comment:
It seems like you could simplify this function to 
   `linkAttributes.?[value] || value`